src/status_im2/common/bottom_sheet/view.cljs

```diff
@@ -200,28 +213,27 @@
                {:style (reanimated/apply-animations-to-style
                         {:opacity bg-opacity}
                         styles/backdrop)}]]
-
-             [gesture/gesture-detector {:gesture bottom-sheet-gesture}
-              [reanimated/view
-               {:style (reanimated/apply-animations-to-style
-                        {:transform [{:translateY translate-y}]}
-                        {:width  window-width
-                         :height window-height})}
-               [rn/view {:style styles/container}
-                (when selected-item
-                  [rn/view {:style (styles/selected-background)}
-                   [selected-item]])
-                [rn/view {:style (styles/background)}
-                 [rn/keyboard-avoiding-view
-                  {:behaviour (if platform/ios? :padding :height)
-                   :style     {:flex 1}}
-                  [rn/view
-                   {:style     (styles/content-style insets)
-                    :on-layout (when-not (and
-                                          (some? @content-height)
-                                          (> @content-height 0))
-                                 on-content-layout)}
-                   children]]
-
-                 (when show-handle?
-                   [rn/view {:style (styles/handle)}])]]]]]))])]))
+             (cond->> [reanimated/view
+                       {:style (reanimated/apply-animations-to-style
+                                {:transform [{:translateY translate-y}]}
+                                {:width  window-width
+                                 :height window-height})}
+                       [rn/view {:style styles/container}
+                        (when selected-item
+                          [rn/view {:style (styles/selected-background)}
+                           [selected-item]])
+                        [rn/view {:style (styles/background)}
+                         [rn/keyboard-avoiding-view
+                          {:behaviour (if platform/ios? :padding :height)
+                           :style     {:flex 1}}
+                          [rn/view
+                           {:style     (styles/content-style insets)
+                            :on-layout (when-not (and
+                                                  (some? @content-height)
```

Comment:
This looks like it can be simplified, I think
at least `(when-not (pos? @content-height) ...)`, you are just checking if it's `0` or `nil?` right?

Reply:
I usually prefer to avoid inverse logic, also `some?` is usually not needed in Clojure, because Clojure has *truthiness* (non-nil is true).

```clojure
(when (and @content-height (<= @content-height 0))
  on-content-layout)
```

Or if we are sure the content height can't be negative:

```clojure
(when (and @content-height (zero? @content-height))
  on-content-layout)
```